2016-04-08.md

```diff
@@ -0,0 +1,37 @@
+We are thrilled to announce that Tessel 2 has shipped to all domestic and international pre-orders.
+
+To our knowledge, this is the first time a distributed, unpaid team of open source contributors has shipped a physical product. This is a major milestone in the open source hardware movement.
+
+Huge thanks to everyone who has contributed so far to the Tessel Project. Whether you're a core contributor, someone who found a typo to fix, or one of our test subjects for the start experience: thank you. We value what you've done.
+
+### "This sounds like a cool project, how can I get involved?"
```

Comment:
No to the first: it is just for team members. We don't want 100 people adding themselves to the Team

Will incorporate the second